pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -479,6 +495,16 @@ func isPDBFromOldVMI(vmi *virtv1.VirtualMachineInstance, pdb *v1beta1.PodDisrupt
 	return ownerRef != nil && ownerRef.UID != vmi.UID
 }
 
+func isPDBFromOldController(pdb *v1beta1.PodDisruptionBudget) bool {
+	// The pdb might be from an old migration-controller that used to create 2-pdbs per migration
+	if pdb == nil {
+		return false
+	}
+
+	_, isOld := pdb.ObjectMeta.Labels["kubevirt.io/migrationName"]
```

Comment:
You could also look at the pdb's name in combination with the label.

So if name has the generated prefix `kubevirt-migration-pdb-` and the migration label exists, then delete.

If you did it this way, you wouldn't have to change the migration label name at all in types.go, which would hopefully be more likely to prevent a future collision